src/Log/Log_Actions.php

```diff
@@ -0,0 +1,55 @@
+<?php declare(strict_types=1);
+
+namespace Tribe\Libs\Log;
+
+use Psr\Log\LoggerInterface;
+
+/**
+ * Log data using WordPress actions.
+ *
+ * @example do_action( Log_Actions::WARNING, 'Something is wrong!', [ 'additional data' ] );
+ */
+class Log_Actions {
+
+	public const EMERGENCY = 'tribe/log/emergency';
+	public const ALERT     = 'tribe/log/alert';
```

Comment:
This is a lot of levels. Do we want to cull this down? In my head, critical = emergency. Alert = info. Noticed WP-CLI doesn't support all these levels as well. So should we match supported WP-CLI log levels?

Reply:
No, these are all required to fulfill the [psr/log contract](https://github.com/php-fig/log/blob/1.1.4/Psr/Log/LoggerInterface.php). 

WP-CLI doesn't have all these levels because it's old and doesn't conform to any standard.  You can use any of these, as the [author just maps them to the next best Log Level](https://github.com/mhcg/monolog-wp-cli/blob/master/src/Monolog/Handler/WPCLIHandler.php#L203-L235).

Using in WP-CLI is a bonus, but the power of this will be logging to [properly supported Monolog libraries](https://github.com/Seldaek/monolog/blob/main/doc/02-handlers-formatters-processors.md#handlers).

Reply:
👍